versioned_docs/version-latest/farming-&-staking/farming/advanced-cli/cli-tips.mdx

```diff
@@ -86,16 +86,15 @@ In general you should be able to download the latest release, and re-start the N
 
 There are some cases where version updates will cause issue with your Node & Farmer and you may have to wipe your node, typically when errors occur. If you have any issues you can always check our [Forums](https://forums.subspace.network) and hop in our [Discord](https://discord.gg/subspace-network) Server to ask for help.
 
-### Wipe & Purge
+### Wipe
```

Comment:
Let's revert those changes, for 3g the preferred method of removing the data is still through `purge-chain`

Reply:
But, we can indeed remove the message "It does not matter if the node/farmer executable is the previous one or from the new snapshot, both will work :)" as it's no longer true.

Reply:
Fixed with latest commit [65b91bd](https://github.com/subspace/subspace-docs/pull/426/commits/65b91bd957b150c423dca0683b25fae3f558b31c)